code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -2270,6 +2270,14 @@ datum
 			result_amount = 4
 			mix_phrase = "The solution yields an astringent powder."
 
+		rubbing_alcohol
+			name = "Rubbing Alcohol"
```

Comment:
might be just me but the chem name (isopropanol) and the name of the recipe should probably be the same